test/functional/test_runner.py

```diff
@@ -463,21 +463,18 @@ def check_script_prefixes():
     """Check that no more than `EXPECTED_VIOLATION_COUNT` of the
        test scripts don't start with one of the allowed name prefixes."""
     EXPECTED_VIOLATION_COUNT = 77
+    LEEWAY = 10
 
-    good_prefixes = ["feature_", "interface_", "mempool_", "mining_", "p2p_", "rpc_", "wallet_", "example_test"]
-    good_prefixes_re = re.compile("|".join(good_prefixes))
-
+    good_prefixes_re = re.compile("(feature|interface|mempool|mining|p2p|rpc|wallet|example)_")
```

Comment:
yeah, this is nicer. Perhaps place `example` at the front to maintain alphabetic sorting